### PR TITLE
Uniform iframes for embedding video/youtube links.

### PR DESCRIPTION
**Pull request by Gemba**
Looks massive but in essence it is the uniform usage of `<iframe ...>` for yt links, using the `youtube-nocookie` site.

I left  the `Bluetooth-Controller.md` page as-is regarding any youtube links as  it needs an overhaul in general.

Some minor issues also addressed:
-  Removal of trailing spaces
-  Some tables fixed in layout

### CODE REVIEW

File: docs/RetroArch-Configuration.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ If you want to edit the entries in the .cfg file for your controller, you will n
 
 On the other hand, maybe you are not sure if the values in the .cfg file is correct or the file is missing entries for buttons that are available on your own controller, such as a "Home" button.  You can run _jstest_ (joystick test) in the terminal by selecting _Quit EmulationStation_ (a keyboard will be required for the following steps).
 
-In the terminal, type and enter
```
Comment (cmitu) [inline docs/RetroArch-Configuration.md:101]: Don't delete the whitespaces here.

File: docs/Sega-CD.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ The Sega CD was an add-on to the Sega Mega Drive/Genesis. It was released in 199
 | [lr-picodrive](https://github.com/libretro/picodrive) | segacd | .bin .chd .cue .iso .m3u | us_scd1_9210.bin, eu_mcd1_9210.bin, jp_mcd1_9112.bin | /opt/retropie/configs/segacd/retroarch.cfg |
 
 ## Emulators: [lr-genesis-plus-gx](https://github.com/libretro/Genesis-Plus-GX), [lr-picodrive](https://github.com/libretro/picodrive)
-**lr-genesis-plus-gx** is recommended for the Pi 2/3 as it has better accuracy. 
+**lr-genesis-plus-gx** is recommended for the Pi 2/3 as it has better accuracy.
 
 ## ROMS
-Accepted File Extensions: **.bin .chd .cue .iso**, **.m3u** for multi-disc games.
```
Comment (cmitu) [inline docs/Sega-CD.md:17]: Don't delete the whitespaces here.

**Review by cmitu — CHANGES_REQUESTED** [review]
Thank you for the changes. Included are more than what's advertised, but they're mostly good changed.

I have a couple of observations:

* Whitespaces matter at the end of the line - 4 whitespaces will force line break without starting a new paragraph. I added a few of those throughout the docs and I'd like to keep them and so I marked the places where they should be reverted. 

* The video `iframe` should have the `allow` attribute with at least `fullscreen` available, otherwise you can't make the embedded video fullscreen. I see you replaced some of the existing `iframe` video tags and stripped the `allow` tag - please restore it and add it also to the other added `iframes`.

**Comment by Gemba** [comment]
Ah, thanks for pointing out. Reverted the 4 spaces and had a close look at the changes again.
The iframes do contain all previous attributes again.

File: docs/Sega-CD.md (hunk 2 above)
Comment (Gemba) [reply docs/Sega-CD.md:17]: Thanks. Done.

**Review by cmitu — APPROVED** [review]
Looks good !
Thank you for the additions and changes.

**Pull request merged**